puppet/modules/web/files/deploy-yumrepo.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+set -xe
+# This is a forced SSH command - uncomment to test locally
+set -f -- $SSH_ORIGINAL_COMMAND
+
+prepcache() {
+	if [[ -e $REPO_PATH ]]; then
+		if [[ $MERGE == false ]] && [[ $OVERWRITE == false ]] ; then
```

Comment:
this was `== true || == true` before, and I think that was wrong (the original code has `unless`, not `if`)

@ekohl would you mind double checking me?

Reply:
Yes, I think you're right and it makes more sense. If you read the error message, we should change `or` to `and` though.

Reply:
Yepp, true